topology/topology.go

```diff
@@ -89,6 +82,13 @@ func (t *TopologyProvider) NetworkTopology() (NetworkTopology, error) {
 		return NetworkTopology{}, err
 	}
 
+	h := sha256.New()
+	h.Write(body)
+	eh := h.Sum(nil)
+	if hash != "" && string(eh) != hash {
```

Comment:
Are you sure string(eh) will work? I assume that the hash should be hex string.

Reply:
Already changed.